setup.py

```diff
@@ -27,9 +27,8 @@
 # set runtime libraries
 runtime_library_dirs = []
 extra_link_args = []
-if platform.system() == 'Linux':
-    runtime_library_dirs.append(libdir)
-elif platform.system() == 'Darwin':
+
+if platform.system() in ['Darwin', 'Linux']:  # now both Linux and Darwin work like this:
```

Comment:
`runtime_library_dirs` is supposed to do exactly that, but I suppose as long as there a platforms or system where it does not produce the right behavior, we should really define it explicitly here, like you did.